tests/unit/test_plots.py

```diff
@@ -28,9 +26,9 @@ def test_plots_order(tmp_dir, dvc):
             name="stage2",
         )
 
-    assert get_files(dvc.plots.show()) == [
```

Comment:
We should have not have used that here, as it was supposed to test the order of `show` results.